Server/src/services/user.service.ts

```diff
@@ -71,10 +71,7 @@ export default class UserService {
 	 */
 	public static async updateUsername(username: string, usernameChangeRequest: UsernameChangeRequest): Promise<boolean> {
 		// Check if input was valid
-		if (username !== usernameChangeRequest.oldUsername) {
-			return false
-		}
-		if (usernameChangeRequest.newUsername !== "") {
+		if (username !== usernameChangeRequest.oldUsername || usernameChangeRequest.newUsername !== "") {
```

Comment:
This doesn't seem right. Why do we return `false` if the new username is **not** `""`
I would guess that `(...) || usernameChangeRequest.newUsername === ""` is more correct